### ISSUE TITLE
Issue

### ISSUE THREAD

**Issue by milqnomadu** [xKeNcHii/Mockup-Automation-Script#3]
https://stackoverflow.com/questions/77591593/photoshop-script-batch-smart-object-insert-replace-issues

**Comment by schroef** [comment]
How is this an issue?
You simply link to another website without any information and just a generic title. This will not help the developper understand your "issie"

**Comment by milqnomadu** [comment]
You're right Schroef, here's the text it did not add for some reason: I have an issue with the code, I will link the detailed explaination because I cannot add images here:

https://stackoverflow.com/questions/77591593/photoshop-script-batch-smart-object-insert-replace-issues

**Comment by schroef** [comment]
I've already commented there ;)
